monitoring/grafana/alerts.yml

```
@@ -0,0 +1,248 @@
+          alert_type: "security_anomaly"
+        annotations:
+          summary: "Banana AI authentication failures high"
+          description: "Authentication failure rate: {{ $value }} failures/sec"
+          runbook_url: "https://docs.banana-ai.com/runbooks/security-analysis"
+
+# Alert manager configuration for Slack/Email notifications
+alerting:
+  alertmanagers:
+    - static_configs:
+        - targets:
+          - alertmanager:9093
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)